checklist.md

```diff
@@ -0,0 +1,25 @@
+# Check Your Code Against the Following Points
+
+## Code Style
+
+1. Use descriptive and correct variable names.
+
+Good example:
+
+```python
+def get_full_name(first_name: str, last_name: str) -> str:
+    return f"{first_name} {last_name}"
+```
+
+Bad example:
+```python
+def get_full_name(x: str, y: str) -> str:
+    return f"{x} {y}"
+```
+
```

Comment:
unnecessary here, add point about don't create unnecessary variables
ex:
```
def test_():
    x = func()
    assert x == [10]
```
 
 would be better
 
```
 def test_():
    assert func() == [10]
```